src/Windows/Avalonia.Direct2D1/Media/Imaging/WicBitmapImpl.cs

```diff
@@ -53,17 +56,38 @@ public WicBitmapImpl(ImagingFactory factory, Stream stream)
         /// <param name="factory">The WIC imaging factory to use.</param>
         /// <param name="width">The width of the bitmap.</param>
         /// <param name="height">The height of the bitmap.</param>
-        public WicBitmapImpl(ImagingFactory factory, int width, int height)
+        /// <param name="pixelFormat">Pixel format</param>
+        public WicBitmapImpl(ImagingFactory factory, int width, int height, APixelFormat? pixelFormat = null)
```

Comment:
That's for consistency with [WritableBitmap.ctor](https://github.com/AvaloniaUI/Avalonia/pull/903/files#diff-b5070a10b906143832bc216ca08b83bdR15) and [CreateWritableBitmap](https://github.com/AvaloniaUI/Avalonia/pull/903/files#diff-bae6bd8ba7fde4a36d352d35193f2648R66). So we could just pass `null` through the call chain and don't worry about default pixel format outside of `WicBitmapImpl`.